tests/Oryx.Integration.Tests/Node/NodeDynamicInstallationTest.cs

```diff
@@ -123,6 +123,56 @@ public async Task CanBuildAndRunApp_UsingScriptCommandAsync(string nodeVersion)
                 });
         }
 
+        [Theory]
+        [InlineData(NodeVersions.Node18Version), Trait("category", "node-18")]
+        public async Task CanBuildAndRunApp_UsingScriptCommand_WithBullseyeBasedImages(string nodeVersion)
+        {
+            // Arrange
+            var appName = "webfrontend";
```

Comment:
I ran some sample apps with the node 18 sdks, and am seeing some of them error out: https://github.com/pauld-msft/acr-test/actions/runs/2891143403

Do you know why that could be happening? Is it a case that we need to support (create react app)

Reply:
Looks related to `https://github.com/webpack/webpack/issues/14532`

Reply:
Looks that some of sample Nodejs apps are not compatible with Node 18. For this issue the latest comments are saying that `webpack ` will require `crypto md5` instead of `md4`? This could be tricky. The `webfrontend` sample apps are working fine with Node 18 but other apps may have different dependencies incompatible with Node 18.

Reply:
Are react apps and angular apps all failing for Node 18, or only certain apps? We'll need to make sure Node 18 will work for react and angular apps in most of cases.

Reply:
sorry, completely missed this. I only tested with the react app and angular app in our sample apps repo